engine/src/timing.rs

```diff
@@ -0,0 +1,41 @@
+use std::time::Instant;
+
+use cimvr_engine_interface::FrameTime;
+
+pub struct Timing {
+    init: Instant,
+    last_frame: Instant,
+    time: FrameTime,
+}
+
+impl Timing {
+    pub fn init() -> Self {
+        let init = Instant::now();
+        Self {
+            last_frame: init,
+            init,
+            time: FrameTime {
+                delta: 0.,
+                time: 0.,
+            },
+        }
+    }
+
+    /// Begin the frame, as far as this clock is concerned.
+    pub fn frame(&mut self) {
```

Comment:
From what I'm reading, the frame function feels like it should have more documentation. After reading the code the comment "Begin the frame, as far as this clock is concerned." I'm confused as to the actual purpose of this function. 

Does this function return the elapsed FrameTime? If so, we should consider changing the name of the function.

Regardless, please add slightly more in depth documentation here :)

Reply:
Fair enough lol